src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, partnerName, sex, isMarried } = person;
+  const FEMALE = 'f';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        { `My name is ${name}` }
+      </h2>
+
+      {
+        age
+          ? (
+            <p className="Person__age">
+              {`I am ${age}`}
+            </p>
+          )
+          : null
+      }
+
+      <p className="Person__partner">
+        {
+          isMarried
+            ? `${partnerName} is my ${sex === FEMALE
+              ? 'husband'
+              : 'wife'
+            } `
+            : `I am not married`
```

Comment:
1. bad practice to use ternary inside other ternary
2. move this logic to func

Reply:
Thanks!